lib/carrierwave/storage/app.rb

```diff
@@ -61,11 +61,10 @@ def zip_file
       end
 
       def tmp_file_path
-        tmpfile = Tempfile.new(["#{app_dir.basename}-", '.zip'])
-        path = tmpfile.path
-        tmpfile.unlink
+        t = Time.now.strftime("%Y%m%d")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.